core/src/main/java/org/apache/iceberg/MetadataUpdate.java

```diff
@@ -328,6 +328,11 @@ public void applyTo(TableMetadata.Builder metadataBuilder) {
     }
   }
 
+  /**
+   * @deprecated since 1.8.1, will be removed in 2.0.0; Use {@link MetadataUpdate.RemoveSnapshots}
```

Comment:
```suggestion
   * @deprecated since 1.10.0, will be removed in 1.11.0; Use {@link MetadataUpdate.RemoveSnapshots}
```
In case this makes it into 1.9.0, the javadoc should be updated to 1.9.0 / removal = 1.10.0